projects/aca-content/src/lib/components/dl-custom-components/name-column/name-column.component.html

```diff
@@ -1,25 +1,30 @@
-<div
-  class="aca-custom-name-column"
-  [ngClass]="{
-    'aca-name-column-container': isFile && isFileWriteLocked
-  }"
->
-  <span
-    role="link"
-    tabindex=0
-    [attr.aria-label]="
-      (isFile ? 'CUSTOM_NAME_COLUMN.ACCESSIBILITY.FILE_LINK_ARIA_LABEL' : 'CUSTOM_NAME_COLUMN.ACCESSIBILITY.FOLDER_LINK_ARIA_LABEL')
-        | translate: { name: displayText$ | async }
-    "
-    class="adf-datatable-cell-value"
-    title="{{ node | adfNodeNameTooltip }}"
-    (click)="onLinkClick($event)"
-    (keyup.enter)="onLinkClick($event)"
-  >
-    {{ displayText$ | async }}
-  </span>
+<div class="aca-custom-name-column">
+  <div [ngClass]="{ 'aca-name-column-container': isFile && isFileWriteLocked }">
+    <span
+      role="link"
+      tabindex="0"
+      [attr.aria-label]="
+        (isFile ? 'CUSTOM_NAME_COLUMN.ACCESSIBILITY.FILE_LINK_ARIA_LABEL' : 'CUSTOM_NAME_COLUMN.ACCESSIBILITY.FOLDER_LINK_ARIA_LABEL')
+          | translate : { name: displayText$ | async }
+      "
+      class="adf-datatable-cell-value"
+      title="{{ node | adfNodeNameTooltip }}"
+      (click)="onLinkClick($event)"
+      (keyup.enter)="onLinkClick($event)"
+    >
+      {{ displayText$ | async }}
+    </span>
 
-  <ng-container *ngIf="isFile && isFileWriteLocked">
-    <aca-locked-by [node]="context.row.node"></aca-locked-by>
-  </ng-container>
+    <ng-container *ngIf="isFile && isFileWriteLocked">
+      <aca-locked-by [node]="context.row.node"></aca-locked-by>
+    </ng-container>
+  </div>
+  <div class="aca-name-column-badges">
+    <ng-container *ngFor="let badge of badges">
+      <adf-dynamic-component  *ngIf="badge.component; else iconBadge" [id]="badge.component" [data]="{ node: node }"></adf-dynamic-component>
```

Comment:
Probably can be just 
[data]="{ node }" (no need to repeat the same name at right side

Reply:
Right, changed